src/NuGet.Core/NuGet.Build.Tasks.Console/NuGet.Build.Tasks.Console.csproj

```diff
@@ -57,6 +59,13 @@
     </AssemblyAttribute>
   </ItemGroup>
 
+  <Target Name="PackBuildOutputs">
+    <ItemGroup>
+      <TfmSpecificPackageFile Include="$(TargetPath)" PackagePath="contentFiles\any\$(TargetFramework)\" PackageCopyToOutput="true" />
+      <TfmSpecificPackageFile Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'" Include="$(ProjectRuntimeConfigFilePath)" PackagePath="contentFiles\any\$(TargetFramework)\" PackageCopyToOutput="true" />
```

Comment:
You should use $(NETCoreTargetFramework) from https://github.com/NuGet/NuGet.Client/blob/dev/build/common.project.props#L15.

Reply:
this is running inside the inner build where TargetFramework is derived from TargetFrameworks. This should be correct.

Reply:
It is running in the inner build. It's just best to condition things based on the TargetFramework alias.

Reply:
But it already uses those as TargetFrameworks is set to https://github.com/NuGet/NuGet.Client/blob/dev/build/common.project.props#L17

Reply:
I'm just suggesting the condition be:
```xml
Condition="'$(TargetFramework)' == '$(NetCoreTargetFramework)'" 
```

All of these are set in the inner build :)

Reply:
ah sorry, I misunderstood. Sure I can do that.